src/SOS/Strike/gchist.cpp

```diff
@@ -257,8 +257,8 @@ void GcHistAddLog(LPCSTR msg, StressMsg* stressMsg)
 
 DECLARE_API(HistStats)
 {
-    INIT_API();
-
+    INIT_API(nullptr);
```

Comment:
It looks a little weird to pass nullptr here.  I understand the intent, by not passing the string we avoid the lookup, but my first question when approaching this code is "why are some of these passing the name and others passing nullptr?"

Is it expensive to try calling ExecuteCommand even for things we know aren't implemented in managed code to make it consistent, and to automatically pick up the replacement if one exists?

(Feel free to leave this as-is, just pointing out that this looks a bit odd with surprising behavior depending on nullptr or otherwise.)

Reply:
Yes, this is ugly. I guess it is kind of expensive since we reverse-pinvoke and look up the command in managed code.  Probably not noticeable. I guess there were also native commands that would never be in managed code (but never say never).

Reply:
I pushed some better named macros (and without the explicit nullptr).